lib/ui/home_page.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import '../bloc/contact_interface_bloc.dart';
+import '../utils/constants.dart';
+import '../model/contact.dart';
+import '../utils/text_styles.dart';
+
+class HomePage extends StatelessWidget {
+  final ContactInterfaceBloc bloc;
+
+  HomePage(
+    this.bloc,
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
```

Comment:
Could you add an appBar with a title like 'Contacts'

Reply:
done